core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/SignerLoader.java

```diff
@@ -38,11 +37,7 @@ private SignerLoader() {
     }
 
     public static AwsV4aHttpSigner getSigV4aSigner() {
```

Comment:
nit, because it is SdkInternalApi, but could keep the method name close to the return type - `getAwsV4aHttpSIgner()`